mopidy/audio/actor.py

```diff
@@ -594,9 +594,16 @@ def set_uri(self, uri, live_stream=False):
         else:
             current_volume = None
 
+        flags = 0x02
+        if download:
+            flags += 0x80
```

Comment:
The default flags are `soft-colorbalance+deinterlace+soft-volume+text+audio+video` according to https://gstreamer.freedesktop.org/documentation/playback/playbin.html?gi-language=c - so as this is you are changing more than it seems.

This is hard to read, and typically you build flag values like this with `|` since you are updating a bit mask. Could you define constants:

```
_FLAG_AUDIO = 0x002
_FLAG_SOFT_VOLUME = 0x010
_FLAG_DOWNLOAD = 0x080
```

and then make the code something like:

```
flags = _FLAG_AUDIO | _FLAG_SOFT_VOLUME
if download:
  flags |= _FLAG_DOWNLOAD
```

Reply:
I think Mopidy just the audio flag currently. https://github.com/mopidy/mopidy/blob/develop/mopidy/audio/actor.py#L22

Reply:
@adamcik I don't remember this changing either, but see https://github.com/mopidy/mopidy/commit/4bbd847a6a508b5bba7668e3847f2e3c721401c5 

We do want the helpfully-named constants though.